sdk/appconfiguration/azure-data-appconfiguration-perf/src/main/java/com/azure/data/appconfiguration/perf/ListConfigurationSettingsTest.java

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.data.appconfiguration.perf;
+
+import com.azure.data.appconfiguration.models.ConfigurationSetting;
+import com.azure.data.appconfiguration.models.SettingSelector;
+import com.azure.data.appconfiguration.perf.core.ServiceTest;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Performs listing multiple configuration settings.
+ */
+public class ListConfigurationSettingsTest extends ServiceTest<PerfStressOptions> {
+    private static final String KEY_PREFIX = "keyPrefix";
+    private static final String SETTING_VALUE = "value";
+
+    private static int settingCount = 1;
+    private static SettingSelector keyFilterSelector = new SettingSelector().setKeyFilter(KEY_PREFIX + "*");
+
+    /**
+     * The ListConfigurationSettingsTest class.
+     *
+     * @param options the configurable options for perf testing this class
+     */
+    public ListConfigurationSettingsTest(PerfStressOptions options) {
+        super(options);
+        settingCount = options.getCount();
+    }
+
+    @Override
+    public Mono<Void> globalSetupAsync() {
+        List<Mono<ConfigurationSetting>> settingMonoList = new ArrayList<>();
+        for (int i = 0; i < settingCount; i++) {
+            settingMonoList.add(configurationAsyncClient.setConfigurationSetting(
+                new ConfigurationSetting().setKey(KEY_PREFIX + i).setValue(SETTING_VALUE)));
+        }
+
+        return Flux.concat(settingMonoList).then();
+    }
+
+    @Override
+    public Mono<Void> globalCleanupAsync() {
+        List<Mono<ConfigurationSetting>> settingMonoList = new ArrayList<>();
+        for (int i = 0; i < settingCount; i++) {
+            settingMonoList.add(configurationAsyncClient.deleteConfigurationSetting(
+                new ConfigurationSetting().setKey(KEY_PREFIX + i).setValue(SETTING_VALUE)));
```

Comment:
Don't you have to delete the key here?

Reply:
deleteConfigurationSetting() deletes the setting in the resource